src/app/shared/components/select/select.component.ts

```diff
@@ -20,6 +20,10 @@ export class SelectComponent<T> implements ControlValueAccessor {
   @Input() public label: string
   @Input() public placeholder?: string
 
+  @Input() public errorMessage?: string
+  @Input() public isRequired?: boolean
+  @Input() public hasError?: boolean
```

Comment:
This attribute should be deleted. Refer to my comment on `TextInputComponent`.

Reply:
Fixed